components/feature/search/src/main/assets/searchplugins/amazon-ca.xml

```diff
@@ -4,9 +4,13 @@
 
 <SearchPlugin xmlns="http://www.mozilla.org/2006/browser/search/">
 <ShortName>Amazon.ca</ShortName>
+<InputEncoding>ISO-8859-1</InputEncoding>
```

Comment:
I realise you're copying from the other Amazons, but this looks wrong as it wouldn't accept utf-8 values. Desktop [changed this a few years back](https://bugzilla.mozilla.org/show_bug.cgi?id=1320712
).

That said, I'm not sure Android is using it - in this repo, I can only [find one use](https://github.com/mozilla-mobile/android-components/blob/d102b22b33efd5ec3324765c7bc26053517c606f/components/feature/search/src/main/java/mozilla/components/feature/search/internal/SearchUrlBuilder.kt#L25), which seems to be enforcing `UTF-8` everywhere.

Reply:
Ah, I did a search on what language does ISO-8859-1 supports and added these to all languages listed.  I'll remove.  Thanks,

Reply:
I am updating the files I've touched to use `<InputEncoding>UTF-8</InputEncoding>`.  Tested with searching `café` and it works well.  Thanks again for pointing this out.